config/initializers/booking_bug_config.rb

```diff
@@ -0,0 +1,24 @@
+module Config
```

Comment:
Our Rails application already has the concept of a config and supports [custom configuration](http://guides.rubyonrails.org/configuring.html#custom-configuration). Why do we need to introduce another way of providing configuration?